src/main/java/racingcar/view/OutputView.java

```diff
@@ -0,0 +1,28 @@
+package racingcar.view;
+
+import camp.nextstep.edu.missionutils.Console;
+import racingcar.domain.Car;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class OutputView {
+    public void displayRoundResults(List<Car> cars) {
+        for (Car car : cars) {
+            System.out.println(car.getName() + " : " + "-".repeat(car.getPosition()));
+        }
+
+    }
+
+    public void displayWinners(List<Car> winners) {
+        if (winners.size() == 1) {
+            System.out.println("최종 우승자 : " + winners.getFirst().getName());
+        } else {
+            String winnerNames = winners.stream()
+                    .map(Car::getName)
+                    .collect(Collectors.joining(", "));
+            System.out.println("최종 우승자 : " + winnerNames);
+        }
```

Comment:
우테코에서 제시하는 [클린코드 pr_checklist](https://github.com/woowacourse/woowacourse-docs/blob/main/cleancode/pr_checklist.md#%EB%AA%A8%EB%93%A0-%EC%9B%90%EC%8B%9C%EA%B0%92%EA%B3%BC-%EB%AC%B8%EC%9E%90%EC%97%B4%EC%9D%84-%ED%8F%AC%EC%9E%A5%ED%96%88%EB%8A%94%EA%B0%80)에 `else`를 사용하지 말라는 사항이 있습니다! 참고하시면 좋을 것 같아요!

저도 1주차에 `else-if`를 사용해서 비슷한 리뷰를 받은 적이 있어요!  `early return`이라는 키워드를 찾아보시면 도움 되실 것 같습니다 👍

Reply:
ㅠㅠ.. 맞습니다.. 이거 나중에 처리하려다가 안 바꿨네요!! 
1명일 때랑 복수일 때를 어떻게 구분하지 하다가 넘겼는데 이렇게 됐군요..

혹시 어떻게 처리하셨나요? 1명일 때와 여러 명일 때??

Reply:
이런식으로 return 해주시면 될 거 같아요!

```suggestion
        if (winners.size() == 1) {
            System.out.println("최종 우승자 : " + winners.getFirst().getName());
            return 
            }
            String winnerNames = winners.stream()
                    .map(Car::getName)
                    .collect(Collectors.joining(", "));
            System.out.println("최종 우승자 : " + winnerNames);
```